agent/engine/dockerclient/dockerclientfactory_test.go

```diff
@@ -23,21 +23,21 @@ import (
 	"github.com/golang/mock/gomock"
 )
 
+const expectedEndpoint = "expectedEndpoint"
+
 func TestGetDefaultClientSuccess(t *testing.T) {
 	ctrl := gomock.NewController(t)
 	defer ctrl.Finish()
 
 	mockClient := mock_dockeriface.NewMockClient(ctrl)
 	mockClient.EXPECT().Ping()
 
-	expectedEndpoint := "expectedEndpoint"
-
 	newVersionedClient = func(endpoint, version string) (dockeriface.Client, error) {
 		if endpoint != expectedEndpoint {
 			t.Errorf("Expected endpoint %s but was %s", expectedEndpoint, endpoint)
 		}
-		if version != string(defaultVersion) {
-			t.Errorf("Expected version %s but was %s", defaultVersion, version)
+		if version != string(getDefaultVersion()) {
```

Comment:
Since you're in this file...want to change it to use `github.com/stretchr/testify/assert`?

```go
assert.Equal(t, string(getDefaultVersion()), version)
```